forc/src/cli/commands/plugins.rs

```diff
@@ -18,15 +22,62 @@ pub struct Command {
     describe: bool,
 }
 
+fn get_file_name(path: &Path) -> String {
+    if let Some(Some(path_str)) = path.file_name().map(|path_str| path_str.to_str()) {
+        path_str.to_owned()
+    } else {
+        path.display().to_string()
+    }
+}
+
 pub(crate) fn exec(command: PluginsCommand) -> ForcResult<()> {
     let PluginsCommand {
         print_full_path,
         describe,
     } = command;
 
+    let mut plugins = crate::cli::plugin::find_all()
+        .map(|path| print_plugin(path.clone(), print_full_path, describe).map(|info| (path, info)))
```

Comment:
I know that `print_plugin` is not introduced here, but seeing that it does not print but format the string, it might make sense to rename it?